core/src/main/java/com/scalar/db/storage/cassandra/SelectStatementHandler.java

```diff
@@ -112,6 +117,16 @@ private Select prepare(Scan scan) {
     return select;
   }
 
+  private Select prepare(ScanAll scanAll) {
+    Select select = getSelect(scanAll);
```

Comment:
It doesn't seem to be doing "ALLOW FILTERING"; thus, simple "scan everything" works but "scan with some predicates" doesn't work?
Please add tests for scan with some predicate that doesn't include a partition key.

https://www.datastax.com/blog/allow-filtering-explained

Reply:
Sorry, as discussed, I misunderstood the spec for this PR. So, forget it!